### ISSUE TITLE
backend: Implement DB import resource

### ISSUE THREAD

**Issue by cghislai** [Valuya/comptoir#135]

**Comment by ymajoros** [comment]
I think we already have that. I already did a couple of imports, so I guess I coded id ;-)

**Comment by cghislai** [comment]
Yep, and the frontend has an UI for it, but the REST resource is a mock currently.

**Comment by ymajoros** [comment]
There is some real resource, really.

Le mer. 23 mars 2016 11:56, cghislai <notifications@github.com> a écrit :

> Yep, and the frontend has an UI for it, but the REST resource is a mock
> currently.
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/Valuya/comptoir/issues/135#issuecomment-200295553>
>
-- 
Yannick Majoros
Valuya sprl

**Comment by cghislai** [comment]
I see /company/{companyId}/import which accepts a dump but do nothing than returning the company ref.
I also just discovered /import/{companyId}/{backendName} which seems to import from a local database on the server.
I am missing something?

**Comment by ymajoros** [comment]
The latter is the right one and does it all. No dumps.

Le mer. 23 mars 2016 16:40, cghislai <notifications@github.com> a écrit :

> I see /company/{companyId}/import which accepts a dump but do nothing than
> returning the company ref.
> I also just discovered /import/{companyId}/{backendName} which seems to
> import from a local database on the server.
> I am missing something?
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/Valuya/comptoir/issues/135#issuecomment-200400054>
>
-- 
Yannick Majoros
Valuya sprl
